src/test/java/de/telran/bank/AccountTest.java

```diff
@@ -54,4 +54,19 @@ void shouldCheckIfANumberIsEven() {
         // then
         Assertions.assertTrue(isEven);
     }
+
+    @Test
+    void shouldGetAccountBalance() throws Exception {
+        mockMvc.perform(MockMvcRequestBuilders.get("/account/balance")
+                        .queryParam("accountId","3")
+                        .queryParam("accountType","checking"))
+                .andExpect(content().string("The balance for accountId = 3 is 1235 on checking"));
+    }
+    @Test
+    void shouldGetAccountBalanceFor1() throws Exception {
```

Comment:
Method should be removed to `shouldGetDefaultAccountBalance`

Reply:
Ok